eospac-wrapper/eospac_wrapper.cpp

```diff
@@ -184,7 +196,17 @@ bool eosSafeInterpolate(EOS_INTEGER *table, EOS_INTEGER nxypairs, EOS_REAL xVals
       std::cerr << "y = " << yVals[i] << std::endl;
     }
   }
-#endif                          // SINGULARITY_EOSPAC_SKIP_EXTRAP
+#endif // SINGULARITY_EOSPAC_SKIP_EXTRAP
+
+  for (int i = 0; i < nopts; i++) {
+    eos_ResetOption(table, &options[i], &errorCode);
+    if (errorCode != EOS_OK) {
+      eos_GetErrorMessage(&errorCode, errorMessage);
+      std::cerr << "Table " << tablename << ":" << std::endl;
+      std::cerr << "eos_ResetOption ERROR " << errorCode << ": " << errorMessage
+                << std::endl;
+    }
+  }
```

Comment:
Didn't measure that, but my guess it is smaller than setting ALL options to a defined state before each call, which would be necessary if you don't reset them right away. Otherwise, you can't rely on the option values, given that you don't know what other EOSPAC calls have happened before each function.

Reply:
While the table handle is associated with the EOSPAC class, how do we know im which order the available functions are called? At a minimum we need to set all options that potentially could have been set by any member function to avoid interference.

Reply:
Ok, as an example: if you look at the usage of the `PofRT_table_`, the calls happen sometimes with `F_CONVERT` and sometimes with `X_Y_PASSTHROUGH` options passed. I can not predict in what order the different EOS functions are called, so I have to make sure the options are in a defined state.

Reply:
Ok I see. Let's leave this for now then. Maybe worth putting a TODO to talk with the EOSPAC folks about though. If setting options implies a performance hit we might want to redesign the class to avoid having to do so.